cmd/helios-cli/main_test.go

```diff
@@ -12,7 +12,6 @@
```

Comment:
Reading into a fixed 256-byte buffer risks truncation. Suggest `io.ReadAll(r)` (or a bufio reader) and comparing the full output. Also consider asserting `strings.HasPrefix(out, "helios ")` to lock the contract.